src/ui/popup/theme/controls/theme-control.tsx

```diff
@@ -1,6 +1,6 @@
 import {m} from 'malevic';
 
-export default function ThemeControl(props: {label: Malevic.Child}, control: Malevic.Child) {
+export default function ThemeControl(props: {label: Malevic.Child}, ...control) {
```

Comment:
If you think there should be an array, then rename the argument to `controls` with type `Malevic.Child[]`.